.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+language: ruby
+script: bin/rspec
+rvm:
+  - 1.9.3
+  - 2.1
+branches:
+  only:
+    - master
+builder_args: --jobs 7
+notifications:
+  irc:
+    channels:
+      - 'chat.freenode.net#chef'
```

Comment:
We don't do any notifications to IRC for other cookbooks, this should be removed.